rpc/send.go

```diff
@@ -261,3 +272,33 @@ func sendOne(client *Client, timeout time.Duration, method string,
 		}
 	}()
 }
+
+// localCall invokes the specified method directly.
+func localCall(s *Server, method string, args proto.Message, done chan *rpc.Call) {
+	s.mu.RLock()
+	m := s.methods[method]
+	s.mu.RUnlock()
+
+	if m.handler == nil {
+		done <- &rpc.Call{
+			Error: newRPCError(util.Errorf("rpc:couldn't find method: %s", method)),
+		}
+		return
+	}
+
+	if _, err := security.CheckRequestUser(args, m.public); err != nil {
+		done <- &rpc.Call{
+			Error: err,
+		}
+		return
+	}
+
+	m.handler(args, func(reply proto.Message, err error) {
```

Comment:
Pretty sure this code isn't used by `server/raft_transport.go`. AFAICT, `rpc.Send` is only used by `kv.DistSender`.